contracts/platform/FundRequestContract.sol

```diff
@@ -143,6 +143,22 @@ contract FundRequestContract is Owned, ApproveAndCallFallBack {
         return "_".strConcat(str);
     }
 
+    //required to be able to migrate to a new FundRequestContract
+    function migrateTokens(address _token, address newContract) external onlyOwner {
+        require(newContract != address(0));
+        if(_token == ETHER_ADDRESS) {
+            newContract.transfer(address(this).balance);
+        } else {
+            ERC20 token = ERC20(_token);
+            token.transfer(newContract, token.balanceOf(address(this)));
+        }
+    }
+
+    //required should there be an issue with available ether
```

Comment:
I'm assuming this shouldn't happen...?

Reply:
shouldn't happen indeed, but if we somehow made a mistake somewhere, we wouldn't be able to claim something with more eth than the balance anymore, as we wouldn't have a way to deposit eth.

Reply:
👍